dolweb/downloads/templates/downloads-index.html

```diff
@@ -87,6 +87,14 @@
 </table>
 </div>
 
+<div id="Other-Linux">
```

Comment:
Can you rename the id to "other-linux" for consistency with other ids on that page?